todo_app/templates/index.html

```diff
@@ -22,7 +22,7 @@ <h1 class="rounded-t bg-slate-800 p-5 text-xl font-semibold text-gray-100">
       </button>
     </form>
     <div class="flex flex-wrap justify-center gap-3">
-      {% for item in items %}
+      {% for item in view_model.items %}
```

Comment:
Nothing wrong with this, but worth noting that since we're not using the new `completed_items` and `todo_items` properties then they very much exist purely as an exercise requirement, i.e. we have test code targeting methods that never actually get executed - so fine for learning purposes as long as we acknowledge that would be bad practice in reality!

Reply:
🖖 acknowledged